src/app/features/itineraries/utils/mapCaseToCardProps.tsx

```diff
@@ -11,20 +11,20 @@ import { Case, ItineraryItem } from "../../types"
 import to from "../../shared/routing/to"
 
 export const casesToCardCaseProps = (cases: Case[], itinerary?: Components.Schemas.Itinerary, addDistance: boolean = false) => {
-  if (!itinerary || !cases) {
+  if (!cases) {
```

Comment:
Shouldn't this be `if (cases.length === 0)` or `if (!cases.length)`?

Reply:
Changed typing, cases could also be undefined.